tests/e2e/cluster_utils.py

```diff
@@ -0,0 +1,87 @@
+"""Utilities for interacting with the OpenShift cluster."""
+
+import subprocess
+
+
+def create_user(name: str):
+    """Create a service account user for testing."""
+    try:
+        subprocess.run(
```

Comment:
prob. refactor this repeated command (all the keyword args are the same)

Reply:
refactor pushed in commit `pavel1`, please see if that's what you meant